src/openforms/config/utils.py

```diff
@@ -24,3 +35,38 @@ def verify_clamav_connection(host: str, port: int, timeout: int) -> "ClamAVStatu
         return ClamAVStatus(can_connect=True)
 
     return ClamAVStatus(can_connect=False, error=result)
+
+
+def _create_csp_settings(obj: Model, directive: str, value: str) -> None:
+    from .models import CSPSetting
+
+    CSPSetting.objects.filter(
+        content_type=get_content_type_for_model(obj), object_id=str(obj.id)
+    ).delete()
+
+    CSPSetting.objects.create(content_object=obj, directive=directive, value=value)
+
+
+def create_digid_eherkenning_csp_settings(
```

Comment:
please move these specific helpers into their respective apps.

Additionally, since `CSPSetting` is our own model, I think that `_create_csp_settings` should be a model manager method:

```py
CSPSetting.objects.set_for(obj, [("form-action", "https://*.digid.nl ..."])
```

the current implementation only ever allows a single `CSPSetting` for a given configuration object since you allow only a single `directive` to be passed. Calling the helper again for another directive will delete the first once through the GFK.